lib/Runtime/Language/AsmJs.cpp

```diff
@@ -580,10 +580,12 @@ namespace Js
                 {
                 case MathBuiltin::Function:{
                     auto func = mathBuiltin.u.func;
+#ifdef ENABLE_SCRIPT_DEBUGGING
                     if (func->GetName() != nullptr)
                     {
                         OutputMessage(m.GetScriptContext(), DEIT_ASMJS_FAILED, _u("Warning: Math Builtin already defined for var %s"), func->GetName()->Psz());
                     }
+#endif
```

Comment:
RaiseMessageToDebugger is already guarded we are missing trace due to this. I think its not needed?